src/MultiValueEvents.tsx

```diff
@@ -50,9 +60,22 @@ export class MultiValueEvents {
         this.refresh(values);
         this._fired++;
         const formService = await WorkItemFormService.getService();
-        formService.setFieldValue(this.fieldName, values.join(";"));
+
+        await formService.setFieldValue(this.fieldName, values.join(";"));
+
         return new Promise<void>((resolve) => {
             this._onRefreshed = resolve;
         });
     }
+    private async _checkFieldType(): Promise<string> {
+        const formService = await WorkItemFormService.getService();
+        const inv = await formService.getInvalidFields();
+        if (inv.length > 0 && inv.some((f) => f.referenceName === this.fieldName)) {
+            const field = await getClient().getField(this.fieldName);
+            if (field.isPicklist && VSS.getConfiguration().witInputs.Values) {
+                return `Set the field ${this.fieldName} to use suggested values rather than allowed values`;
```

Comment:
nit - do we have field display name to use? Consider a case like non-english field name it can be really weird since we auto gen that :)

Reply:
Good point

Reply:
I think we have the name but the it is not localized.